docs/setup/settings.asciidoc

```diff
@@ -73,7 +73,7 @@ error messages.
 
 [[logging-verbose]]`logging.verbose:`:: *Default: false* Set the value of this setting to `true` to log all events, including system usage information and all requests. Supported on Elastic Cloud Enterprise.
 
-`logging.useUTC`:: *Default: true* Set the value of this setting to `false` to log events using the timezone of the server, rather than UTC.
+`logging.timezone`:: *Default: UTC* Set to the canonical timezone id to log events using that timezone.
```

Comment:
Can we link here? https://en.wikipedia.org/wiki/List_of_tz_database_time_zones

And provide an example like `US/Pacific`?

I am considered that folks will not easily understand what the "canonical timezone id" they need to provide would be.

Reply:
Added with 9629036.